dist/.goreleaser.yaml

```diff
@@ -97,12 +98,15 @@ nfpms:
       compression: xz
     overrides:
       deb:
+        file_name_template: "{{ .ProjectName }}-server_{{ .Version }}.{{ .Os }}_{{ .Arch }}"
         scripts:
           preinstall: deb/scylla-manager-server.preinst
           postinstall: deb/scylla-manager-server.postinst
           preremove: deb/scylla-manager-server.prerm
           postremove: deb/scylla-manager-server.postrm
       rpm:
+        replacements:
+          amd64: x86_64
```

Comment:
Does `goreleaser` actually generate `.rpm` files with `amd64` in the name? That's sad.

We probably need something for `aarch64` too?

Reply:
The default is to generate the rpms with amd64. again we need it to be identical to what we have in Scylla

I guess we need to address ARM as well, but i think it could be in another PR , what do you think?

Reply:
Yeah, better to defer, since we don't even have `arm64` in the `goarch` section.

Reply:
well it's supported based on https://stackoverflow.com/questions/20728767/all-possible-goos-value